### DISCUSSION TITLE
[Feature] Basic editing

### DISCUSSION THREAD

**Discussion by phellarv** [immich-app/immich#4028] [Feature Request]
### The feature

I would very much like to see very basic editing added to Immich.
Rotate, Crop and Mirror would be enough to satisfy 95% of my editing needs. The rest I will download, edit and upload to fix.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by danieldietzler** [comment]
There are already people working on this in #3271 :)
